src/Psy/ExecutionLoop/Loop.php

```diff
@@ -21,7 +21,7 @@
 class Loop
 {
     /**
-     * The non-forking loop doesn't have much use for Configuration :)
+     * The non-forking loop doesn't have much use for Configuration :).
```

Comment:
So you mean to waste my time on this only because it doesn't likes you? ah, nice point, then feel fre to replace it yourself